package.json

```diff
@@ -48,6 +48,7 @@
     "lodash": "4.17.21",
     "lru-cache": "9.1.1",
     "mime": "3.0.0",
+    "mime-types": "2.1.35",
```

Comment:
We already have [mime](https://www.npmjs.com/package/mime), it seems that it's similar to [mime-types](https://www.npmjs.com/package/mime-types). I think if possible we should decide for one of the two.